lib/deploy/hosting/hashcache.js

```diff
@@ -36,6 +36,10 @@ exports.dump = function(cwd, name, data) {
     count++;
     st += path + "," + data[path].mtime + "," + data[path].hash + "\n";
   }
-  fs.outputFileSync(cachePath(cwd, name), st, { encoding: "utf8" });
-  logger.debug("[hosting] hash cache [" + name + "] stored for", count, "files");
+  try {
+    fs.outputFileSync(cachePath(cwd, name), st, { encoding: "utf8" });
+    logger.debug("[hosting] hash cache [" + name + "] stored for", count, "files");
+  } catch (e) {
+    logger.debug("[hosting] unable to store hash cache [" + name + "]");
```

Comment:
This should print the `e.stack` in the debug message so it's clear what's actually happening.